sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql

```diff
@@ -2,6 +2,7 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.active_users_aggregates`
 AS
 SELECT
-  *
+  *,
+  `mozfun.norm.os`(os) AS normalized_os
```

Comment:
Since the field 'os' is already [collecting the normalized os](https://github.com/mozilla/bigquery-etl/blob/d01ce2fc8a6193121dc961951c850cf4bf8bcc3e/sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/query.sql#L18), should we call this field **os_grouped** same as in the the [Look-954](https://mozilla.cloud.looker.com/looks/954) to differentiate the contents?

Reply:
Yeah, it can be confusing. My recommendation aims for the final users to easily identify what's in there.